jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -265,6 +266,19 @@ public void setConnectors(Connector[] connectors)
             _connectors.addAll(Arrays.asList(connectors));
     }
 
+    /**
+     * Add a {@link Connection.Listener} as a bean to all connectors on the server, this
+     * will register the listener with all connections accepted by the connectors.
+     * @param listener the listener to be added.
```

Comment:
@gregw changing this to `addBeanToAllConnectors` will make this javadoc irrelevant.

Maybe it is worth keeping it as a `Listener` method to keep this documentation around, it doesn't seem obvious that adding a `Listener` as a bean on a connector would add it for all new connections. Only other place I can see this mentioned is the javadoc for `ConnectionStatistics` which doesn't seem the right place.

Reply:
See my suggestion for the javadoc below.